build-tools/packages/build-tools/tsconfig.json

```diff
@@ -16,6 +16,6 @@
 		"moduleResolution": "node16",
 		"types": ["node"],
 	},
-	"include": ["src/**/*.ts"],
+	"include": ["src/**/*.ts", "src/test/mockPackage.json"],
```

Comment:
This seems wrong... why is it needed?

Reply:
I thought I needed the mockPackage.json to build but I was able to just inline the short json in the test.